--- gen_trtllm_dockerfile.py	2023-10-13 11:34:40.139364775 -0700
+++ gen_trtllm_dockerfile_new.py	2023-10-13 11:57:13.801664683 -0700
@@ -43,12 +43,11 @@
     if clone_repo:
         df += """
 # FIXME: Update the url
-RUN git clone --single-branch --depth=1 -b {} https://{}:{}@gitlab-master.nvidia.com/ftp/tekit_backend.git tensorrtllm_backend
+RUN git clone --single-branch --depth=1 -b {} https://github.com/triton-inference-server/tensorrtllm_backend.git
 RUN cd tensorrtllm_backend && git submodule update --init --recursive
 RUN cp tensorrtllm_backend/tensorrt_llm/docker/common/install_tensorrt.sh /tmp/
 RUN rm -fr tensorrtllm_backend
-    """.format(trtllm_be_repo_tag, os.environ["REMOVE_ME_TRTLLM_USERNAME"],
-               os.environ["REMOVE_ME_TRTLLM_TOKEN"])
+    """.format(trtllm_be_repo_tag)
     else:
         df += """
     COPY tensorrt_llm/docker/common/install_tensorrt.sh /tmp/
