--- CMakeLists.txt	2023-10-09 22:16:21.813010695 -0700
+++ CMakeLists.txt.new	2023-10-09 22:17:55.846645690 -0700
@@ -261,12 +261,27 @@
            "${TRTLLM_DIR}/cpp/build/tensorrt_llm/libtensorrt_llm_static.a")

 add_library(tensorrt_llm_batch_manager STATIC IMPORTED)
-set_property(
-  TARGET tensorrt_llm_batch_manager
-  PROPERTY
-    IMPORTED_LOCATION
-    "${TRTLLM_DIR}/cpp/build/tensorrt_llm/batch_manager/libtensorrt_llm_batch_manager_static.a"
-)
+execute_process(
+  COMMAND ${Python3_EXECUTABLE} "-c"
+          "import torch; print(torch.compiled_with_cxx11_abi(),end='');"
+  RESULT_VARIABLE _PYTHON_SUCCESS
+  OUTPUT_VARIABLE USE_CXX11_ABI)
+message(STATUS "USE_CXX11_ABI: ${USE_CXX11_ABI}")
+if(USE_CXX11_ABI)
+  set_property(
+    TARGET tensorrt_llm_batch_manager
+    PROPERTY
+      IMPORTED_LOCATION
+      "${TRTLLM_DIR}/cpp/tensorrt_llm/batch_manager/libtensorrt_llm_batch_manager_static.a"
+  )
+else()
+  set_property(
+    TARGET tensorrt_llm_batch_manager
+    PROPERTY
+      IMPORTED_LOCATION
+      "${TRTLLM_DIR}/cpp/tensorrt_llm/batch_manager/libtensorrt_llm_batch_manager_static.pre_cxx11.a"
+  )
+endif()

 add_library(nvinfer_plugin_tensorrt_llm SHARED IMPORTED)
 set_property(
