diff --git a/inflight_batcher_llm/CMakeLists.txt b/inflight_batcher_llm/CMakeLists.txt
index 3f0df3b..a9aaa1e 100644
--- a/inflight_batcher_llm/CMakeLists.txt
+++ b/inflight_batcher_llm/CMakeLists.txt
@@ -197,7 +197,7 @@ add_library(tensorrt_llm STATIC IMPORTED)
 set_property(TARGET tensorrt_llm PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/../tensorrt_llm/cpp/build/tensorrt_llm/libtensorrt_llm_static.a")

 add_library(tensorrt_llm_batch_manager STATIC IMPORTED)
-set_property(TARGET tensorrt_llm_batch_manager PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/../tensorrt_llm/cpp/build/tensorrt_llm/batch_manager/libtensorrt_llm_batch_manager_static.a")
+set_property(TARGET tensorrt_llm_batch_manager PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/../tensorrt_llm/cpp/tensorrt_llm/batch_manager/libtensorrt_llm_batch_manager_static.pre_cxx11.a")


 add_library(nvinfer_plugin_tensorrt_llm SHARED IMPORTED)
 set_property(TARGET nvinfer_plugin_tensorrt_llm PROPERTY IMPORTED_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/../tensorrt_llm/cpp/build/tensorrt_llm/plugins/libnvinfer_plugin_tensorrt_llm.so")
